### ISSUE TITLE
&apos; vs #039;

### ISSUE THREAD

**Issue by michaeltorbert** [semperfiwebdesign/all-in-one-seo-pack#965]
We're using the former to escape apostrophe's, whereas the single quote seems more common. 
Not sure if the information here is outdated: https://stackoverflow.com/questions/2083754/why-shouldnt-apos-be-used-to-escape-single-quotes

Interestingly, WordPress, with no plugins active, escapes the blog title and post title differently.

<img width="496" alt="screen shot 2017-07-15 at 9 48 54 am" src="https://user-images.githubusercontent.com/17364998/28239836-18cec4aa-6943-11e7-8f29-56e638b3f22d.png">

**Comment by wpsmort** [comment]
See also the OG meta, as referenced in issue #968

**Comment by amostajo** [comment]
[PR:970](https://github.com/semperfiwebdesign/all-in-one-seo-pack/pull/970)

**Comment by wpsmort** [comment]
@amostajo @michaeltorbert I tested PR #970 and found that on Posts only, the og:title and twitter:title are using the character code &#8217; for the auto generated title.  For example:
```
<meta property="og:title" content="Steve&#8217;s Post Title | Steve&#039;s Test Site" />
```
All other meta used the character code &#039;

This is the same behavior as v2.3.13.2 so it should not hold up release of this bug fix.

**Comment by amostajo** [comment]
@wpsmort I was not able to reproduce this behavior, I think this got solved in one of the PRs already merged. On my end everything gets converted to `&#039;`. Can you please double check?

**Comment by wpsmort** [comment]
I have tested and confirmed that this is fixed in the current development branch.

**Comment by michaeltorbert** [comment]
@wpsmort When did this get fixed? If we decided that it was broken in the last version, it couldn't have magically fixed itself in the dev branch which is the same as the live version.

**Comment by wpsmort** [comment]
@michaeltorbert This got fixed by PR #980.  I tested commits prior to that and confirmed it was broken.  Testing of tPR #980 confirmed that this fixed the issue.
